massa-execution-worker/src/execution.rs

```diff
@@ -686,7 +686,7 @@ impl ExecutionState {
 
         // send `roll_price` * `roll_count` coins from the sender to the recipient
```

Comment:
If you take to modify the first comment I made it would be nice to change this source code comment which is totally wrong even if it's not related to this PR.